com.onesignal.unity.android/Runtime/AndroidUserManager.cs

```diff
@@ -93,7 +104,32 @@ public void RemoveSms(string sms)
             => _user.Call("removeSms", sms);
 
         public void Initialize() {
+            _user.Call("addObserver", new InternalUserChangedHandler(this));
             _pushSubscription.Initialize();
         }
+
+        private sealed class InternalUserChangedHandler : OneSignalAndroidJavaProxy {
+            private AndroidUserManager _parent;
+
+            public InternalUserChangedHandler(AndroidUserManager userManager) : base("user.state.IUserStateObserver") {
+                _parent = userManager;
+            }
+
+            /// <param name="state">UserChangedState</param>
+            public void onUserStateChange(AndroidJavaObject state) {
+                UnityEngine.Debug.Log("onUserStateChange fired");
```

Comment:
I don't think this debug log is needed since the other events don't have logs. We could add debug logs to all the other events instead. What do you think?